app/containers/Toast.js

```diff
@@ -20,17 +19,19 @@ const styles = StyleSheet.create({
 	}
 });
 
-export const LISTENER = 'Toast';
+const getToastRef = toast => this.toast = toast;
+
+export const showToast = ({ message }) => {
```

Comment:
Yes, I thought to do that also. but didn't find a way to export the class Toast. Is that possible ?? since I am unable to export the class since its wrapped across the **withTheme** which is accepting a component

Reply:
@diegolmello @djorkaeffalexandre I need little help unable to figure out the way you are saying :)

Reply:
![Screen Shot 2020-10-29 at 3 53 57 PM](https://user-images.githubusercontent.com/29778115/97619455-0075b100-19ff-11eb-9d64-c111707d6763.png)
![Screen Shot 2020-10-29 at 3 54 06 PM](https://user-images.githubusercontent.com/29778115/97619465-023f7480-19ff-11eb-8b14-341af62b7cbf.png)
:top: This code works as expected, IMO it's better

Reply:
I have done this previously but didn't get as expected.. @djorkaeffalexandre  is there any error ??

![up1](https://user-images.githubusercontent.com/44283521/97661025-41220880-1a99-11eb-87a4-00c098ce428f.png)
![up3](https://user-images.githubusercontent.com/44283521/97661039-48491680-1a99-11eb-96ff-0c3c488c1de3.png)
![up](https://user-images.githubusercontent.com/44283521/97661047-4bdc9d80-1a99-11eb-9fe4-f5fe2f1e7675.png)